man/fmidi-seq.1

```diff
@@ -0,0 +1,16 @@
+.TH FMIDI\-SEQ "1" "August 2021" "" "User Commands"
+.SH NAME
+fmidi\-seq \(en print the complete sequence from a MIDI file
```

Comment:
> complete sequence

This tool is in fact a sequencer, it calculates the precise times in seconds where individual midi events would occur in a playback. The result is output in text, as tuples: track no, time, event